extensions/crc/src/daemon-commander.ts

```diff
@@ -44,8 +44,14 @@ export class DaemonCommander {
       method: 'GET',
     };
 
-    const { body } = await got(options);
-    return JSON.parse(body);
+    try {
+      const { body } = await got(options);
+      return JSON.parse(body);
+    } catch (error) {
```

Comment:
Maybe it's better to log error somehow, just to be able detect error flow?

Reply:
error is now logged as suggested